frontend/src/scenes/dashboard/DashboardHeader.tsx

```diff
@@ -120,19 +120,33 @@ export function DashboardHeader(): JSX.Element | null {
             <PageHeader
                 buttons={
                     dashboardMode === DashboardMode.Edit ? (
-                        <LemonButton
-                            data-attr="dashboard-edit-mode-save"
-                            type="primary"
-                            onClick={() => setDashboardMode(null, DashboardEventSource.DashboardHeader)}
-                            tabIndex={10}
-                            disabled={dashboardLoading}
-                        >
-                            Done editing
-                        </LemonButton>
+                        <>
+                            <LemonButton
+                                data-attr="dashboard-edit-mode-discard"
+                                type="secondary"
+                                onClick={() =>
+                                    setDashboardMode(null, DashboardEventSource.DashboardHeaderDiscardChanges)
+                                }
+                                tabIndex={9}
+                            >
+                                Discard Changes
```

Comment:
This does not discard changes to the dashboard's layout – they are saved immediately! That will appear like a bug to a user. It's okay to merge the current mechanics, as they allow for new possibilities worth the small chance of confusion, but a follow-up PR should also align layout editing with the Cancel/Save workflow (we can maybe put it off though – your call)

Reply:
This should discard changes to the layout. I disabled saving the layout changes instantly from [here](https://github.com/PostHog/posthog/pull/24303/files#diff-749ecf967fe174bf1f31fd4d01262d1335dda1ad07cf98b5f618e936e06ff781L919)
In my updated [PR desc video](https://github.com/PostHog/posthog/pull/24303#issue-2459761010), I show how the layout changes reset when you click 'Cancel'. But there might be a bug if it still doesn't work for you? 😓